### PR TITLE
More robust way of detecting the mod_macro vhosts

### PR DESCRIPTION
**Pull request by joohoi**
Work towards #1224

### CODE REVIEW

**Comment by bmw** [comment]
Hit by letsencrypt/boulder#756. Here's the [log](https://gist.github.com/04ed91d8bb0b920ad6a8). Kicking Travis.

**Comment by pde** [comment]
Probably just an incorrectly implemented XSS... move along, nothing to see :)

File: letsencrypt-apache/letsencrypt_apache/configurator.py [hunk 1]
```diff
@@ -459,7 +440,12 @@ def _create_vhost(self, path):
         filename = get_file_path(path)
         is_enabled = self.is_site_enabled(filename)
 
-        vhost = obj.VirtualHost(filename, path, addrs, is_ssl, is_enabled)
+        macro = False
+        if "/macro/" in path.lower():
```
Comment (pde) [inline letsencrypt-apache/letsencrypt_apache/configurator.py]: Perhaps `macro = ("/macro/" in path.lower())`?

Or `macro = True if ("/macro" in path.lower()) else False`?

**Comment by pde** [comment]
My search for vertical minimalism is not that necessary.  Merging :)

**Pull request merged**